src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessSink.cs

```diff
@@ -66,7 +67,13 @@ public class ExceptionlessSink : ILogEventSink {
         public ExceptionlessSink(Func<EventBuilder, EventBuilder> additionalOperation = null, bool includeProperties = true, ExceptionlessClient client = null) {
             _additionalOperation = additionalOperation;
             _includeProperties = includeProperties;
-            _client = client ?? ExceptionlessClient.Default;
+            if (client != null) {
+                _client = client;
+            }
+            else {
+                _client = ExceptionlessClient.Default;
+                _client.Configuration.UseLogger(new SelfLogLogger());
```

Comment:
But in this case we will overwrite logging if user have already set up logging for the client. 
Maybe we should apply logging if logger is null.